### DISCUSSION TITLE
Charging w/ Thunderbolt 3 USB-C port - does it work?

### DISCUSSION THREAD

**Discussion by hangrydave** [valnoxy/t480-oc#73] [Q&A] [answered]
Question is basically in the title, but my main charging port on my ThinkPad is dodgy, and the connection is flaky when I wiggle the power cable. Currently, when using Windows, I charge using the USB-C port on the Thunderbolt 3 thingy. Does this method of charging work with Mac OS?

**Comment by cchewanik** [comment, marked as answer]
Just tried it, works fine.

**Reply by hangrydave** [reply]
Great, thank you!
